public/index.html

```diff
@@ -6,6 +6,14 @@
     <title>Document</title>
 </head>
 <body>
+    <header><form>
+        <input type="text" placeholder="Username">
+        <button type="submit">Enter Username</button>
+    </form></header>
+    <h2>Welcome to teacher o matic</h2>
+    <p>Enter your user name in the header</p>
+
```

Comment:
Can this be placed in a wrapper?

```suggestion
<div class="wrapper">
    <h2>Welcome to teacher o matic</h2>
    <p>Enter your user name in the header</p>
</div>

```